ihatemoney/tests/main_test.py

```diff
@@ -368,11 +364,11 @@ def test_api_project_creation_does_not_need_captcha(self):
                 "contact_email": "raclette@notmyidea.org",
             },
         )
-        self.assertTrue(resp.status, 201)
```

Comment:
Is this assert right at the origin ? assertTrue take only one parameter, second is an optional msg.

Did we want to assert resp.status == 201 here or something else ?

Reply:
Good catch. This is fixed.